src/consts/config.ts

```diff
@@ -2,6 +2,7 @@ import { ADDRESS_BLACKLIST } from './blacklist';
 
 const isDevMode = process?.env?.NODE_ENV === 'development';
 const version = process?.env?.NEXT_PUBLIC_VERSION || '0.0.0';
+const registryUrl = process?.env?.NEXT_PUBLIC_REGISTRY_URL || undefined;
```

Comment:
nice
does NEXT_ prefix really make sense here?

Reply:
The `NEXT_PUBLIC_` prefix isn't for taste. It's what allows this env var to be exposed to client-side :)